vip-cache-manager/ttl-manager.php

```diff
@@ -27,6 +27,16 @@ function enforce_rest_api_read_ttl( $response, $rest_server, $request ) {
 		return $response;
 	}
 
+	// Don't override existing Cache-Control headers sent via PHP
+	$php_headers = headers_list();
+	foreach ( $php_headers as $header ) {
+		if ( 0 === stripos( $header, 'Cache-Control' ) ) {
+			return $response;
+		}
+	}
+	unset( $header );
```

Comment:
We only need to explicitly `unset()` if the value is passed by reference in the loop.

Reply:
Done.